Assets/Plugins/Source/Editor/Configs/ConfigFieldAttribute.cs

```diff
@@ -65,12 +65,17 @@ public enum ConfigFieldType
         /// A list of strings.
         /// </summary>
         TextList,
+
+        Button,
```

Comment:
Can this be given a comment?

Reply:
Yup! So added. Marking this conversation as resolved.